### PR TITLE
Add raw values operation

### PR DESCRIPTION
**Pull request by ronensc**

### CODE REVIEW

**Review by eranra — APPROVED** [review]
Looks good to me ... no comments ... only request is to add also **documentation in README** so that it is well understood that choosing `raw_values` is different from the rest of the operations and that the values are added to the "recent raw values" field and not to the regular recent op value ... also give some explanation why this is needed.

**Comment by ronensc** [comment]
@KalmanMeth @eranra Please pay attention to the added documentation. Is it clear enough?

File: README.md [hunk 1]
```diff
@@ -482,6 +482,31 @@ parameters:
...
+- `total_count`: the total count
+- `recent_raw_values`: a slice with the raw values of the recent batch
+- `recent_op_value`: the aggregate value of the recent batch
+- `recent_count`: the count of flowlogs in the recent batch
+
+These fields are used by the next stage (for example `prom` encoder).
+The pipeline processes flowlogs in batches.
+The output fields with `recent_` prefix are related to the recent batch.
+They are needed when exposing metrics in Prometheus using Counters and Histograms.
+Prometheus Counters API accepts the amount to be added to the counter and not the final value as in Gauges.
```
Comment (eranra) [inline README.md:501]: - I suggest to add the word `delta`
- I suggest replacing `final value` with `total value`

Comment (eranra) [inline README.md:501]: - I suggest to add the word `delta`
- I suggest replacing `final value` with `total value`

File: README.md [hunk 2]
```diff
@@ -482,6 +482,31 @@ parameters:
...
+In this case, `recent_op_value` and `recent_count` should be used as the `valuekey`.
+The API of Histograms accepts the sample value, so it could be added to the right bucket.
```
Comment (eranra) [inline README.md:503]: s/right/appropriate

Comment (eranra) [inline README.md:503]: s/right/appropriate

**Review by eranra — APPROVED** [review]
Minor updates to doc .... looks good to me. LGTM

**Pull request merged**